### DISCUSSION TITLE
Pre releases

### DISCUSSION THREAD

**Discussion by miguelsousa** [fonttools/fontbakery#3876] [General]
@felipesanches FYI, I was the one how pushed this pre-release tag [v0.8.10rc](https://github.com/googlefonts/fontbakery/releases/tag/v0.8.10rc).
Did it because I need to test the latest changes via PyPI install.

**Comment by felipesanches** [comment]
That's good! Once I finish work on #3837, I'll create a new release and publish it. Feel free to tag release candidates like this one, if that's useful for you, but please leave to me the final decision to make the actual releases.

But of course you can bug me whenever you feel that a new release is taking too long to happen, and I'll try to do it even if it means postponing a set of issues not addressed yet (except any critical one that might need to be addressed urgently).

**Reply by miguelsousa** [reply]
> please leave to me the final decision to make the actual releases

Sure will.

> But of course you can bug me whenever you feel that a new release is taking too long to happen

Will do. Thanks!
